<!doctype html>
<html>
  <head>
  
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>EffectorPredictor  by matteobrilli</title>
    <link rel="stylesheet"  href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <script src="javascripts/scale.fix.js"></script>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <ul>
          <li><a href="https://github.com/matteobrilli/EffectorPredictor/releases/download/v1.0/EP_v1.0.zip"> Download <strong> EffectorPredictor </strong></a></li>
          <li><a href="https://github.com/matteobrilli/EffectorPredictor/releases/download/v1.0/EffectorPredictor_manual_v1.0.pdf.zip"> Download the <strong>  Manual </strong></a></li>
          <li><a href="https://github.com/matteobrilli/EffectorPredictor/releases/download/v1.0/datasetChlamydophila.zip">Download <strong> C. pneumoniae </strong> Example </a></li>
       </ul>
      
                              <p align="left"><a href="URL"><img src="https://cloud.githubusercontent.com/assets/10756420/7987313/07845f8c-0add-11e5-871a-3bd18796b61d.png" height="60%" width="60%" /></a></p>

        <p class="view"><a href="https://github.com/matteobrilli/EffectorPredictor">View the Project on GitHub <small>matteobrilli/EffectorPredictor</small></a></p>
        
        </header>
        <section>
        <center>
        <h1>EffectorPredictor </h1>
        <p>EffectorPredictor is a pipeline for virulence - related gene prediction from one or multiple genomes.</p>
          </center>
        <h3>
<a id="what-it-is" class="anchor" href="#what-it-is" aria-hidden="true"><span class="octicon octicon-link"></span></a>What it is</h3>

<p>EffectorPredictor (EP) is a pipeline combining several pieces of software with the aim of returning a reliable identification of positively selected genes from pathogen genomes. After a stringent clustering of protein sequences, EffectorPredictor performs state of the art positive selection analysis on all the clusters. Additional tools are run to provide an output integrating different sources of information, such as presence of Pfam domains and signal peptide for secretion.
A less stringent clustering is also run to provide a classification into homologous groups and to identify proteins similar to known effectors if these sequences are provided at the beginning of the analysis, or simply to aggregate the clusters for which positive selection was calculated. Each homologous cluster is also used as an input to Meme to identify recurrent proteins motifs, as it is known that effectors are often characterized by those signatures. These motifs are used to scan all the sequences for additional occurrences and the data is reported as matlab compatible files for additional, manual analysis.</p>

<a id="Run-with-test-data" class="anchor" href="#Run-with-test-data" aria-hidden="true"><span class="octicon octicon-link"></span></a>Run with test data</h3>

<p> To run the paper's dataset of Chlamydophila, extract all files in a folder, and follow the instructions to prepare EP for run. Then Run EP from the folder where the dataset has been stored. A pre-compiled blast is also provided to shorten the analysis that will nonetheless take a few hours.</p>

<p align="center"><a href="URL"><img src="https://cloud.githubusercontent.com/assets/10756420/5959695/9adc2762-a7d0-11e4-8b7a-51d92615337d.png" height="80%" width="80%" /></a></p>

<h3>
<a id="java-dependencies" class="anchor" href="#java-dependencies" aria-hidden="true"><span class="octicon octicon-link"></span></a>Java dependencies</h3>
<ul>
<li>biojava3-core-3.0.7.jar:<a href="http://biojava.org/wiki/BioJava:Download_3.0.5"> Download Biojava3</a></li>
<li> commons-math3-3.3.jar: <a href="http://commons.apache.org/proper/commons-math/download_math.cgi"> Download Math3</a></li>
<li> jung-api-2.0.1.jar: <a href="http://sourceforge.net/projects/jung/files/">Download Jung-api</a></li>
<li> Colt: this library is no more maintained, can be found for download here <a href="https://code.google.com/p/d-p2p-sim/downloads/detail?name=colt.jar&can=2&q=">Download colt.jar</a></li>
</ul>
<h3>
<a id="third-party-required-software" class="anchor" href="#third-party-required-software" aria-hidden="true"><span class="octicon octicon-link"></span></a>Third-party required software</h3>
<ul>
Several programs are launched by EffectorPredictor (EP) and therefore they need to be installed on your machine; their path must be added to the EP configuration file, see the manual.

<li> Blast <a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastDocs&DOC_TYPE=Download">Go</a></li>
<li> MCL <a href="http://www.micans.org/mcl/index.html?sec_software"> Go </a></li>
<li> signalP <a href="http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp"> Go </a></li>
<li> HH-suite <a href="ftp://toolkit.genzentrum.lmu.de/pub/HH-suite/"> Go </a></li>
<li> Muscle <a href="http://www.drive5.com/muscle/"> Go </a></li>
<li> RevTrans <a href="http://www.cbs.dtu.dk/services/RevTrans/download.php"> Go </a></li>
<li> PhyML <a href="http://www.atgc-montpellier.fr/phyml/binaries.php"> Go </a></li>
<li> codeml (PAML)<a href="http://abacus.gene.ucl.ac.uk/software/paml.html"> Go </a></li>
<li> Meme <a href="http://meme.nbcr.net/meme/downloads.html"> Go </a></li>
</ul>
<h3>
<a id="citation" class="anchor" href="#citation" aria-hidden="true"><span class="octicon octicon-link"></span></a>Citation</h3>

<h3>
<a id="support-or-contact" class="anchor" href="#support-or-contact" aria-hidden="true"><span class="octicon octicon-link"></span></a>Support or Contact</h3>

<p><a href="mailto:matteo.brilli@fmach.it">matteo.brilli@fmach.it</a></p>


  </body>
      </section>
    </div>
    <footer>
      <p>Project maintained by <a href="https://github.com/matteobrilli">matteobrilli</a></p>
      <p>Hosted on GitHub Pages &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></p>
    </footer>
    <!--[if !IE]><script>fixScale(document);</script><![endif]-->
       <!-- hitwebcounter Code START -->
<a href="http://www.hitwebcounter.com/countersiteservices.php" target="_blank">
<img src="http://hitwebcounter.com/counter/counter.php?page=5973244&style=0035&nbdigits=5&type=ip&initCount=0" title="Efficient stats" Alt="Efficient stats"   border="0" >
</a><br/>
<!-- hitwebcounter.com --><a href="http://www.hitwebcounter.com/" title="site meter statistics" 
target="_blank" style="font-family: Geneva, Arial, Helvetica, sans-serif; 
font-size: 11px; color: #97878C; text-decoration: underline ;"><code>site meter statistics</code>
</a>   

<!-- hitwebcounter Code START -->
<a href="http://www.hitwebcounter.com/countersiteservices.php" target="_blank">
<img src="http://hitwebcounter.com/counter/counter.php?page=5973243&style=0035&nbdigits=5&type=page&initCount=0" title="Efficient stats" Alt="Efficient stats"   border="0" >
</a><br/>

<!-- hitwebcounter.com --><a href="http://www.hitwebcounter.com/" title="site meter statistics" 
target="_blank" style="font-family: Geneva, Arial, Helvetica, sans-serif; 
font-size: 11px; color: #97878C; text-decoration: underline ;"><code>site meter statistics</code>
</a>   
  
  

</html>
